CHANGELOG.md

```diff
@@ -2,6 +2,63 @@
 
 All notable changes to the kdb extension will be documented in this file.
 
+
+# v1.3.0
+
+### Enhancements
+
+- Ability to connect to kdb Insights Enterprise using a WSL connection.
```

Comment:
and SSH

Reply:
@ecmel how do i word this?
- Ability to connect to kdb Insights Enterprise using a WSL and SSH connection.

or 

- Ability to connect to kdb Insights Enterprise using a WSL connection and SSH.

or something different

Reply:
@LizNorris Ability to connect to Insights Enterprise in case of remote development environments such as WSL and SSH.

(Note: We possibly support all kind of remote dev since we use the vscode apis for login now)

Reply:
```suggestion
- Ability to connect to Insights Enterprise in case of remote development environments such as WSL and SSH.
```